src/components/card/card.jsx

```diff
@@ -12,11 +12,22 @@ import * as styles from "./card.module.scss";
 const Card = ({ data }) => {
   const categoryBlog = useCategory(data.frontmatter.category);
 
+  const urlImages = () => {
+    if (typeof window !== 'undefined' && window.location.href === 'http://localhost:8000/') {
```

Comment:
What happen if you use the app in production (the URL isn't different to localhost:8000)

Reply:
Done